src/Plant/photosynthesis/get_leaf_j.jl

```diff
@@ -1,8 +1,10 @@
-# calculate j from par
+"""
+    get_leaf_j(jmax, par)
+This function calculates j from par using a quadratic function
+"""
 function get_leaf_j(jmax::FT, par::FT) where {FT}
-    a =  0.9
-    b = -0.3 * par - jmax
-    c =  0.3 * par * jmax
-    j = ( -b - sqrt(b*b-4*a*c) ) / a * 0.5
+    b = PS_J_QY * par + jmax
+    c = PS_J_QY * par * jmax
+    j = ( b - sqrt(b^NUMB_2 - NUMB_4*PS_J_CR*c) ) / PS_J_CR * NUMB_0_5
```

Comment:
```suggestion
    j = ( b - sqrt(b^2 - 4*PS_J_CR*c) ) / PS_J_CR * FT(0.5)
```
Based on [conversion and promotion](https://docs.julialang.org/en/v1/manual/conversion-and-promotion/#conversion-and-promotion-1) rules, the `2` and `4` will promote to the type `FT`, but the `0.5` must be type-casted to `FT`. This is a nice final form since it does not depend on any global variables, and can therefore be better optimized by the compiler.